src/Microsoft.DotNet.ImageBuilder/src/Commands/BuildOptions.cs

```diff
@@ -29,6 +29,7 @@ public class BuildOptions : DockerRegistryOptions, IFilterableOptions
         public ServicePrincipalOptions ServicePrincipal { get; set; } = new();
         public string? Subscription { get; set; }
         public string? ResourceGroup { get; set; }
+        public IEnumerable<(string Id, string Src)>? SecretInfo { get; set; }
```

Comment:
This can then clean up the calling code to not have to check for null.
```suggestion
        public IEnumerable<(string Id, string Src)> SecretInfos { get; set; } = Enumerable.Empty<(string Id, string Src)>();
```